arch-gulpfile.js

```diff
@@ -130,6 +131,44 @@ function generateServiceWorker() {
     return false;
   }
 }
+
+function inlineCriticalCSS() {
+  const HOST = process.env.HOST || 'localhost';
+  const PORT = process.env.PORT || 3000;
+  const PATH = process.env.CRITICAL_PATH || '/';
```

Comment:
yep it would be great if we can stick the config in app/config/*.json or sw.config.js